editions/tw5.com/tiddlers/widgets/ActionMangleTagsWidget.tid

```diff
@@ -0,0 +1,104 @@
+created: 20150925181827962
+creator: matabele
+modified: 20150925191257169
+modifier: Matabele
+revision: 7
+tags: ActionWidgets Widgets
+title: ActionMangleTagsWidget
+type: text/vnd.tiddlywiki
+
+! Introduction
+
+The ''action-mangletags'' widget is an [[action widget|ActionWidgets]] that manipulates the tags field of the target tiddler. ActionWidgets are used within triggering widgets such as the ButtonWidget.
+
+The "action-mangletags" widget may be used to do any of the following.
+
+* append a tag to the tags field
+* find and remove a tag from the tags field 
+* find and replace a tag in the tags field
+
+! Content and Attributes
+
+|Attribute |Explanation |h
+|$tiddler |Title of the tiddler to manipulate (if not provided defaults to the [[current tiddler|Current Tiddler]] |
+|$add |Optional name of a tag to append to the tags field |
+|$remove |Optional name of a tag to find and remove |
+
+! Examples
```

Comment:
You'll see that the other widget documentation separates the examples out into separately transcluded tiddlers. Would you mind following the same style?